client-v2/src/shared/components/article/Article.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
```

Comment:
I've reorganised these components into their own folders to keep the main directory from becoming cluttered and to colocate related things.